### PR TITLE
Adjust demo notebooks path for guided notebooks PR check

### PR DESCRIPTION
**Pull request by sutaakar**
# Issue link

# What changes have been made

Adjusting demo notebooks path for guided notebooks PR check.
Also fixing broken links in readme.

# Verification steps

Run Guided notebooks PR check.

## Checks
- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] Testing is not required for this change

### CODE REVIEW

File: src/codeflare_sdk/demo-notebooks/guided-demos/2_basic_interactive.ipynb [hunk 1]
```diff
@@ -65,10 +65,10 @@
     "    head_gpus=1, # For GPU enabled workloads set the head_gpus and num_gpus\n",
     "    num_gpus=1,\n",
     "    num_workers=2,\n",
-    "    min_cpus=2,\n",
-    "    max_cpus=2,\n",
-    "    min_memory=8,\n",
-    "    max_memory=8,\n",
+    "    min_cpus='250m',\n",
```
Comment (Fiona-Waters) [inline src/codeflare_sdk/demo-notebooks/guided-demos/2_basic_interactive.ipynb:68]: Are these changes necessary?

Comment (sutaakar) [reply src/codeflare_sdk/demo-notebooks/guided-demos/2_basic_interactive.ipynb:68]: they are needed to make the notebook running in GitHub runner, which has limited CPU and memory

**Review by ChughShilpa — APPROVED** [review]
/lgtm
